### ISSUE TITLE
Venus Next expansion uses "longer game variant" by default

### ISSUE THREAD

**Issue by kevinb9n** [bafolts/terraforming-mars#1553]
Selecting Venus Next does not currently auto-select World Government Terraforming, so by *default* I am playing the "if you want a longer game" variant rule.

I think that the user should have to make an explicit selection if they actually want to play a game variant.

**SOLUTION A**

Any time I select Venus Next, WGT is automatically selected as well. I can deselect if desired.
Any time I deselect Venus Next, WGT is likewise deselected. I can re-select if desired.

Users who don't notice what is happening might accidentally play a "more standard" version of the game than they meant. I think this would err in the right direction. The more nonstandard they like their games, the more familiar they probably will become with the interface.

**SOLUTION B**

The current WGT option is only visible and enabled whenever Venus Next is not on. Whenever VN is on, that WGT option goes away, and a different option appears, called "Skip World Government Terraforming".

I'm not really sure this is better than solution A, it's just more clear about when you are activating a variant.

**Comment by kevinb9n** [comment]
Note solution B seems more amenable to a future development of persistent preferences.

I could store my preference for whether I like to add WGT to non-Venus games, separately from my preference for whether I like to remove it from Venus games. Under solution A, I would end up wanting a three-state preference: {WGT always; WGT never; WGT only when Venus}.

**Comment by nwai90** [comment]
@kevinb9n Makes sense. I will add solution A for now. It's not ideal to introduce a new option that is the opposite of an existing option imo

**Comment by nwai90** [comment]
Implemented in https://github.com/bafolts/terraforming-mars/pull/1551
